TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/RunningActivity.java

```diff
@@ -305,6 +305,8 @@ public void run() {
                 }
             });
             displayToastMessage(R.string.save_map_success);
+            // Restart Tango to be able to load the new map.
+            mTangoServiceClientNode.callTangoConnectService(TangoConnectRequest.RECONNECT);
```

Comment:
No, we restart Tango to restart motion tracking (saving an ADF stops it) and update the list of maps available. Instead of restarting Tango which can be confusing, we could only update the list of maps and add a toast message saying that motion tracking has stopped.